src/views/AddItem/AddItem.jsx

```diff
@@ -19,7 +32,48 @@ export default function AddItem({ token }) {
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    addItem();
+    const checkForErrors = validate();
```

Comment:
I like the separation here it makes it really easy to move around the validate function